kubeflow/jupyter/jupyter-web-app.libsonnet

```diff
@@ -216,6 +216,58 @@
       },
     },
 
+    istioVirtualService:: {
+      apiVersion: "networking.istio.io/v1alpha3",
+      kind: "VirtualService",
+      metadata: {
+        name: params.name,
+        namespace: params.namespace,
+      },
+      spec: {
+        hosts: [
+          "*",
+        ],
+        gateways: [
+          "kubeflow-gateway",
+        ],
+        http: [
+          {
+            match: [
+              {
+                uri: {
+                  prefix: "/" + params.prefix + "/",
```

Comment:
do we want exact or prefix?
also, do we need the trailing /?

Reply:
I tried to keep the same semantics - in ambassador config it's prefix, and it has trailing /